### ISSUE TITLE
Add comments to generated code

### ISSUE THREAD

**Issue by nipunn1313** [dropbox/mypy-protobuf#166]
If there are comments in the original proto files - transpose them into the generated files - to help out developer tools interpreting the .pyi files - or folks directly reading the .pyi files

There's a nice example of this in pb-jelly-gen here
https://github.com/dropbox/pb-jelly/blob/master/pb-jelly-gen/codegen/codegen.py

Nice block comment on SourceCodeInfo in descriptor.proto here
https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.proto#L754

**Comment by dzbarsky** [comment]
@nipunn1313 I implemented this in the go version almost a year ago :) See https://github.com/dropbox/dbx_build_tools/blob/8d3eaa70055985e268ec5bac3351497ec2708d64/go/src/dropbox/build_tools/protoc-gen-mypy/main.go#L269 and callers.

Now that I'm gone, I doubt anyone is updating the go code in this repo; would it make sense to delete it and add a pointer to dbx_build_tools (which does get upstreamed automatically) to the README?

**Comment by nipunn1313** [comment]
yeah - I'm actually looking at migrating rSERVER to use the python one - since it's grown a lot more functionality and is essentially adopted by the community at this point.

I actually filed this as it was one of the handful of things that the go impl did that the python didn't!

There were a couple others I had to implement (well_known_types, and rename self -> self_ when message field named self)

This one doesn't feel like a blocker, but also... shouldn't be too hard!
